adb-server/adb-server-command-types/src/main/java/com/kaspersky/adbserver/commandtypes/AdbCommand.kt

```diff
@@ -2,4 +2,7 @@ package com.kaspersky.adbserver.commandtypes
 
 import com.kaspersky.adbserver.common.api.Command
```

Comment:
Could you also reflect all these cases in tests? I mean testing a single command, a command with arguments and a command with commands in arguments (complex commands with pipes and etc.).